src/card/info-status/info-status-card.html

```diff
@@ -5,18 +5,18 @@
          class="info-img"/>
     <span class="info-icon {{$ctrl.status.iconClass}}"></span>
   </div>
-  <div>
+  <div class="info-content-container">
     <h2 class="card-pf-title" ng-if="$ctrl.status.title">
       <a href="{{$ctrl.status.href}}" ng-if="$ctrl.status.href">
         <span class="">{{$ctrl.status.title}}</span>
       </a>
       <span ng-if="!$ctrl.status.href">
-            <span class="">{{$ctrl.status.title}}</span>
-          </span>
+        <span class="">{{$ctrl.status.title}}</span>
+      </span>
     </h2>
-    <p ng-if="$ctrl.shouldShowHtmlContent" ng-bind-html="$ctrl.trustAsHtml(item)"
-       ng-repeat="item in $ctrl.status.info track by $index"></p>
-    <p ng-if="!$ctrl.shouldShowHtmlContent" ng-bind="item"
-       ng-repeat="item in $ctrl.status.info track by $index"></p>
+    <div ng-if="$ctrl.shouldShowHtmlContent" class="info-content" ng-bind-html="$ctrl.trustAsHtml(item)"
```

Comment:
Where is ".info-content" defined? 👀

Reply:
YOU NOTICED!  its not, used for testing, also just in case user wants to grab each of those rows and do something with them :-/

Reply:
You can leave it, might be nice for apps to use to add settings, how about a rename though? like card-pf-info-item?

Reply:
yeah that's what i was thinking! will rename